### PR TITLE
docs(cockpit): add grafana alloy doc

### PR DESCRIPTION
**Pull request by nerda-codes**

### CODE REVIEW

File: observability/cockpit/how-to/send-metrics-logs-to-cockpit.mdx [hunk 1]
```diff
@@ -95,7 +95,7 @@ You can push logs with any Loki compatible agent such as the [Promtail](https://
   ### Creating a Grafana dashboard and visualizing your Python logs
 
     1. [Log into Grafana](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/) using your [Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/).
-    2. Click the **Home** icon, then click **Dashboards**.
+    2. Click the **Toggle menu** icon, then click **Dashboards**.
```
Comment (RoRoJ) [inline observability/cockpit/how-to/send-metrics-logs-to-cockpit.mdx:98]: If it is an icon and not text that says "Toggle menu" I think it shouldn't be in bold

File: observability/cockpit/how-to/send-metrics-with-grafana-alloy.mdx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        ts=2024-06-27T14:23:13.432896Z level=info msg="Done replaying WAL" component_path=/ component_id=prometheus.remote_write.cockpit subcomponent=rw remote_name=aa0551 url=https://xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx.metrics.cockpit.fr-par.scw.cloud/api/v1/push duration=12.900498616s
+        ```
+
+## Visualizing your metrics
+
+1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
+2. Click **Open dashboards** to open your managed dashboards in Grafana. You are redirected to the Grafana website.
+3. Enter your [Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/).
+4. Click **Log in**. The Grafana overview page displays.
+5. Click the **Toggle menu**.
```
Comment (RoRoJ) [inline observability/cockpit/how-to/send-metrics-with-grafana-alloy.mdx:98]: same here

File: observability/cockpit/quickstart.mdx [hunk 3]
```diff
@@ -64,8 +64,8 @@ In this documentation, we show you how to retrieve your Grafana credentials to a
 <Message type="">
 **Going further** <br /><br />
 
-  If you have not set up any agent yet, check out our [documentation](/observability/cockpit/api-cli/configuring-grafana-agent/) to find out how to configure the Grafana agent, push your data and visualize it in Grafana.
-  You must have created resources to monitor in order to push your data to Cockpit.
+  If you have not set up any agent yet, check out our [documentation](/observability/cockpit/how-to/send-metrics-with-grafana-alloy/) to find out how to configure Grafana Alloy, push your metrics, and visualize them in Grafana.
```
Comment (RoRoJ) [inline observability/cockpit/quickstart.mdx:67]:
```suggestion
  If you have not set up an agent yet, check out our [documentation](/observability/cockpit/how-to/send-metrics-with-grafana-alloy/) to find out how to configure Grafana Alloy, push your metrics, and visualize them in Grafana.
```

**Review by RoRoJ — APPROVED** [review]

File: observability/cockpit/how-to/send-metrics-logs-to-cockpit.mdx (hunk 1 above)
Comment (nerda-codes) [inline observability/cockpit/how-to/send-metrics-logs-to-cockpit.mdx:98]:
```suggestion
    2. Click **Toggle menu**, then click **Dashboards**.
```

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**